#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_fix_infinite_loop.dpatch by Andrew Lee (李健秋) <ajqlee@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: solve the problem on runs configure in infinite loop.

@DPATCH@
diff -urNad lxsession-edit-0.1.1~/configure.in lxsession-edit-0.1.1/configure.in
--- lxsession-edit-0.1.1~/configure.in	2009-07-07 00:05:38.000000000 +0800
+++ lxsession-edit-0.1.1/configure.in	2009-08-15 21:48:34.590642849 +0800
@@ -5,6 +5,9 @@
 AM_CONFIG_HEADER(config.h)
 AM_MAINTAINER_MODE
 
+# Init intltool
+IT_PROG_INTLTOOL
+
 AC_ISC_POSIX
 AC_PROG_CC
 AM_PROG_CC_STDC
